### PR TITLE
Added `None` checking in `min_balance` validation

### PR DESCRIPTION
**Pull request by razmikarm**
Added `None` checking in `min_balance` validating condition so the user can look up for balances with more than 0 amount.

### CODE REVIEW

**Comment by razmikarm** [comment]
Can make more changes like this for keeping general coding style, if you approve this changes.

File: algosdk/v2client/indexer.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def accounts(
             query["limit"] = limit
         if next_page:
             query["next"] = next_page
-        if min_balance:
+        if not min_balance is None:
```
Comment (algochoi) [inline algosdk/v2client/indexer.py:148]: nit: I have a slight preference for the below due to readability

```suggestion
        if min_balance is not None:
```

File: algosdk/v2client/indexer.py [hunk 2]
```diff
@@ -202,7 +202,7 @@ def asset_balances(
...
```
Comment (algochoi) [inline algosdk/v2client/indexer.py:205]: nit
```suggestion
        if min_balance is not None:
```

**Review by algochoi — COMMENTED** [review]
This looks like a good change, thank you! Please feel free to add additional checks.

**Review by algochoi — APPROVED** [review]
Let's merge this after tests pass...

**Comment by algochoi** [comment]
https://github.com/algorand/py-algorand-sdk/pull/507 supercedes this PR.

**Pull request closed without merging**